packages/utils/src/lib/reports/utils.ts

```diff
@@ -112,21 +112,15 @@ export function countCategoryAudits(
   // Create lookup object for groups within each plugin
   const groupLookup = plugins.reduce<Record<string, Record<string, Group>>>(
     (lookup, plugin) => {
-      if (!plugin.groups.length) {
+      if (plugin.groups.length === 0) {
         return lookup;
       }
 
       return {
         ...lookup,
         [plugin.slug]: {
-          ...plugin.groups.reduce<Record<string, Group>>(
-            (groupLookup, group) => {
-              return {
-                ...groupLookup,
-                [group.slug]: group,
-              };
-            },
-            {},
+          ...Object.fromEntries(
+            plugin.groups.map(group => [group.slug, group]),
           ),
         },
```

Comment:
Notice that this spreads into an otherwise empty object.

```ts
[plugin.slug]: {
   ...Object.fromEntries(...),
}
```

Can be simplified:

```ts
[plugin.slug]: Object.fromEntries(...),
```

Reply:
Good point